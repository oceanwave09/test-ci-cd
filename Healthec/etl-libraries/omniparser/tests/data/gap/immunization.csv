~Row~,~Domain~,~Patient First Name~,~Patient Last Name~,~Patient DOB~,~External Patient ID~,~External Practice ID~,~Code~,~Name~,~Dictionary Name~,~Date Performed~,~Date Ordered~,~Date Due~,~Refused~,~Med Unnecessary RSN Code~,~Med Unnecessary RSN Name~,~Med Unnecessary RSN Dict~,~NPI~,~TIN~,~Provider Name~,~Administered Amount~,~Administered Unit~,~Vaccine Lot Number~,~Immu Manufacturer Code~,~Vendor~,~Source Type~,~PATIENT_ID~,~Remove Specified Immu Flag~,~VFC Eligibility Dict Name~,~VFC Eligibility Code~
~1~,~Immunizations~,~First7660601~,~Last7660601~,~19600101~,~7660601~,~Epic~,~100141~,~INFLUENZA SHOT TIV (INJECT)~,~~,~20210120~,~~,~~,~~,~~,~~,~~,~1316998693~,~~,~GHANI, NESRENE A~,~~,~~,~~,~~,~~,~~,~RN7EVM4S7Z7499FJUZLM7RRCM2L2JT8CDGWOQZB2~,~~,~~,~~
~2~,~Immunizations~,~First7660602~,~Last7660602~,~19580101~,~7660602~,~Epic~,~100150~,~INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)~,~~,~20211220~,~~,~~,~~,~~,~~,~~,~1316998693~,~~,~GHANI, NESRENE A~,~~,~~,~199363919~,~~,~~,~~,~IBXEP0B1Y6MBH02112RG567IJZQPF7QEES7KKY5R~,~~,~~,~~
~3~,~Immunizations~,~First7660603~,~Last7660603~,~19920101~,~7660603~,~Epic~,~100150~,~INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)~,~~,~20210118~,~~,~~,~~,~~,~~,~~,~1316998693~,~~,~GHANI, NESRENE A~,~~,~~,~P100242858~,~~,~~,~~,~DN7M0T0SA7U6WWYWSO5VYE7Q2ZFBT5O4JN534OIF~,~~,~~,~~
~4~,~Immunizations~,~First7660604~,~Last7660604~,~19890101~,~7660604~,~Epic~,~100150~,~INFLUENZA VACCINE, INJECTION, QUADRIVALENT (PRESERVATIVE-FREE)~,~~,~20221129~,~~,~~,~~,~~,~~,~~,~1316998693~,~~,~GHANI, NESRENE A~,~~,~~,~A534028~,~~,~~,~~,~S176Y2EGTDK6TACILF4P1TG72BCMXQERYWSNG6KV~,~~,~~,~~
~5~,~Immunizations~,~First7660605~,~Last7660605~,~19690101~,~7660605~,~Epic~,~100207~,~COVID-19 VACCINE (MODERNA)~,~~,~20211105~,~~,~~,~~,~~,~~,~~,~1326010760~,~~,~KADO, HADEEL N~,~~,~~,~939905~,~~,~~,~~,~NCKMK1Y42Y1W2TZIURJ1QI1MG4OY4EN5FA6QMRJU~,~~,~~,~~
