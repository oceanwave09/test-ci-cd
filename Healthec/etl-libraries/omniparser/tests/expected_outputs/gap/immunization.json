[
	{
		"administered_amount": "NESRENE A",
		"code": "100141",
		"date_performed": "20210120",
		"domain": "Immunizations",
		"external_patient_id": "7660601",
		"external_practice_id": "Epic",
		"name": "INFLUENZA SHOT TIV (INJECT)",
		"patient_dob": "b699c2b9c5b08522dc986a2def4aca6e",
		"patient_first_name": "920b1bfa8aeb6237faf53f11643c8ecf",
		"patient_last_name": "41c337237eef08cdc7d9f625231d112a",
		"practice_tin": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_name": "f11616e6643764810c57b634aa499617",
		"provider_npi": "ab51f77da0c9ad2e8bd2ed93c32bcea2",
		"remove_specified_immunization_flag": "RN7EVM4S7Z7499FJUZLM7RRCM2L2JT8CDGWOQZB2",
		"row": "1",
		"row_id": "bf66339c-c21c-45f0-a5a8-0ecfffbf1d9c"
	},
	{
		"administered_unit": "GHANI",
		"code": "100150",
		"date_due": "20211220",
		"date_performed": "QUADRIVALENT (PRESERVATIVE-FREE)",
		"dictionary_name": "INJECTION",
		"domain": "Immunizations",
		"external_patient_id": "7660602",
		"external_practice_id": "Epic",
		"name": "INFLUENZA VACCINE",
		"patient_dob": "56a973e9f56eb206d960f2974ef9804b",
		"patient_first_name": "4369a95df3b5152bdd7c64a72efd1aa6",
		"patient_last_name": "fc65ea507d571ca1dde7d800f2f95573",
		"practice_tin": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_name": "ab51f77da0c9ad2e8bd2ed93c32bcea2",
		"provider_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"row": "2",
		"row_id": "d6b42a5c-c4c5-4cd3-9948-459617a18958",
		"source_type": "199363919",
		"vaccine_lot_number": "NESRENE A",
		"vfc_eligibility_code": "IBXEP0B1Y6MBH02112RG567IJZQPF7QEES7KKY5R"
	},
	{
		"administered_unit": "GHANI",
		"code": "100150",
		"date_due": "20210118",
		"date_performed": "QUADRIVALENT (PRESERVATIVE-FREE)",
		"dictionary_name": "INJECTION",
		"domain": "Immunizations",
		"external_patient_id": "7660603",
		"external_practice_id": "Epic",
		"name": "INFLUENZA VACCINE",
		"patient_dob": "65d8491e60430a0736b2abfd79856d4d",
		"patient_first_name": "0430717827ce4c57e07c1e445bc70477",
		"patient_last_name": "bd39484c0baafaf573a91d7eeb470750",
		"practice_tin": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_name": "ab51f77da0c9ad2e8bd2ed93c32bcea2",
		"provider_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"row": "3",
		"row_id": "5d6616a5-4250-46c6-8184-a3563108c976",
		"source_type": "P100242858",
		"vaccine_lot_number": "NESRENE A",
		"vfc_eligibility_code": "DN7M0T0SA7U6WWYWSO5VYE7Q2ZFBT5O4JN534OIF"
	},
	{
		"administered_unit": "GHANI",
		"code": "100150",
		"date_due": "20221129",
		"date_performed": "QUADRIVALENT (PRESERVATIVE-FREE)",
		"dictionary_name": "INJECTION",
		"domain": "Immunizations",
		"external_patient_id": "7660604",
		"external_practice_id": "Epic",
		"name": "INFLUENZA VACCINE",
		"patient_dob": "9fde4afa3f81b3e57465ff6668b68b41",
		"patient_first_name": "4c2170080510d285b48e195f9c690c5a",
		"patient_last_name": "29244f7cbc96db65581dba3839e472fe",
		"practice_tin": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_name": "ab51f77da0c9ad2e8bd2ed93c32bcea2",
		"provider_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"row": "4",
		"row_id": "127cf113-4163-453e-b32a-3e6efa15d0cb",
		"source_type": "A534028",
		"vaccine_lot_number": "NESRENE A",
		"vfc_eligibility_code": "S176Y2EGTDK6TACILF4P1TG72BCMXQERYWSNG6KV"
	},
	{
		"administered_amount": "HADEEL N",
		"code": "100207",
		"date_performed": "20211105",
		"domain": "Immunizations",
		"external_patient_id": "7660605",
		"external_practice_id": "Epic",
		"immunization_manufacture_code": "939905",
		"name": "COVID-19 VACCINE (MODERNA)",
		"patient_dob": "6d65a93dd15e1ce35deca05d2f90426c",
		"patient_first_name": "fe7030703fd7abee786ea532bf04cd7a",
		"patient_last_name": "4b4e3c02a281e755267bd96cc57327b0",
		"practice_tin": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"provider_name": "32422d6e324a5b78291bf5ce57d22481",
		"provider_npi": "d18643274cd97dafc887ec6e1564e871",
		"remove_specified_immunization_flag": "NCKMK1Y42Y1W2TZIURJ1QI1MG4OY4EN5FA6QMRJU",
		"row": "5",
		"row_id": "407bbc8b-0cc2-417b-bb25-ecfe1a7e2ea4"
	}
]
